meta-ostro/conf/distro/ostro.conf

```diff
@@ -236,6 +236,9 @@ PREFERRED_PROVIDER_virtual/java-initial-native = "cacao-initial-native"
 PREFERRED_PROVIDER_virtual/java-native = "cacao-native"
 PREFERRED_PROVIDER_virtual/javac-native = "ecj-bootstrap-native"
 
+# Java runtime
+PREFERRED_RPROVIDER_java2-runtime = "openjre-8"
```

Comment:
As far as I can tell, they are separate recipes (no DEPENDS or RDEPENDS on jre in jdk recipe), although both use the same .inc files that hold the common portions of the recipes.

https://github.com/ostroproject/ostro-os/blob/master/meta-java/recipes-core/openjdk/openjdk-8_72b05.bb
https://github.com/ostroproject/ostro-os/blob/master/meta-java/recipes-core/openjdk/openjre-8_72b05.bb